<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageOBBDlg</class>
 <widget class="QWidget" name="ImageOBBDlg">
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>289</width>
	<height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageOBBDlg</string>
  </property>
  <widget class="QPushButton" name="pushButtonOBBOK">
   <property name="geometry">
	<rect>
	 <x>210</x>
	 <y>60</y>
	 <width>61</width>
	 <height>31</height>
	</rect>
   </property>
   <property name="text">
	<string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPathOBBSrc">
   <property name="geometry">
	<rect>
	 <x>30</x>
	 <y>30</y>
	 <width>31</width>
	 <height>28</height>
	</rect>
   </property>
   <property name="text">
	<string>src</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditImageOBBDes">
   <property name="geometry">
	<rect>
	 <x>80</x>
	 <y>70</y>
	 <width>113</width>
	 <height>22</height>
	</rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPathOBBDes">
   <property name="geometry">
	<rect>
	 <x>30</x>
	 <y>70</y>
	 <width>31</width>
	 <height>28</height>
	</rect>
   </property>
   <property name="text">
	<string>des</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditImageOBBSrc">
   <property name="geometry">
	<rect>
	 <x>80</x>
	 <y>30</y>
	 <width>113</width>
	 <height>22</height>
	</rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
